staking-cli/tests/cli.rs

```diff
@@ -23,80 +26,9 @@ async fn stake_table_versions(#[case] _version: StakeTableContractVersion) {}
 
 const TEST_MNEMONIC: &str = "wool upset allow cheap purity craft hat cute below useful reject door";
 
-trait AssertSuccess {
```

Comment:
This trait's functionality is no longer needed because `assert_cmd` covers it, in a much better way.